dotcom-rendering/package.json

```diff
@@ -67,6 +59,7 @@
 		"@testing-library/react": "16.3.0",
 		"@testing-library/user-event": "14.6.1",
 		"@types/body-parser": "1.19.2",
+		"@types/chai": "5.2.2",
```

Comment:
> An ambient chai type error has been resolved by adding @types/chai.

It looks like `chai` appears in our dependency tree due to storybook's use of vitest? What was the error you saw that this fixed?